web-ui/src/components/onboard_progress/ProgressIndicator.jsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
```

Comment:
I believe this is an older way of importing React. Now we can just do `import React from 'react';`